rocky/reports/templates/partials/report_names_header.html

```diff
@@ -3,9 +3,19 @@
 
 <h2>{% translate "Report name" %}</h2>
 <p>
-    {% blocktranslate trimmed %}
-        Give your report a custom name and optionally add the reports' reference date
-        to the name. To do so you can select a standard option or use a <a href="https://strftime.org/" target="_blank">Python
-        strftime code</a> in the report name.
-    {% endblocktranslate %}
+    {% if recurrence %}
+        {% blocktranslate trimmed %}
+            Define a custom name format for your report(s). This format will be applied to all
+            generated (sub)reports. To make the report names more descriptive,
+            you can include placeholders for the object name and/or the report type. Use the
+            placeholder "{ooi}" for the object name and "{report type}" for the report type.
+            For example, the format "{report type} for {ooi}" could generate: "DNS Report for example.com".
```

Comment:
We should note that users can also use the strftime placeholders to insert the Referencedate time into the report name at time of generation..